inspector/src/components/actionTabs/tabs/propertyGrids/animations/graphActionsBar.tsx

```diff
@@ -56,7 +56,7 @@ export class GraphActionsBar extends React.Component<IGraphActionsBarProps> {
             onClick={this.props.addKeyframe}
           />
           <IconButtonLineComponent
-            tooltip={'Remove Keyframe'}
+            tooltip={'Frame'}

```

Comment:
Wonder if we should change the tooltip as it looks less descriptive.

Reply:
Yes, the idea for this is to "frame" the selected keyframes... I guess we could have "Frame selected keyframes"